src/main/java/core/mate/academy/model/Machine.java

```diff
@@ -4,16 +4,13 @@
     private String name;
     private String color;
 
-    public String getName() {
-        return name;
-    }
+    public Machine() {
 
-    public void setName(String name) {
```

Comment:
Why do you remove setters and getters?

Reply:
I thought that they aren't used. I'll add them